docs/devel.rst

```diff
@@ -12,9 +12,9 @@ List of dependencies:
 * Dune (>= 1.2.0)
```

Comment:
All this is a duplication of README.md. why to keep it? I think that all the things related to dependencies and compilation should be written in README.md only.

Reply:
I agree